flow/network.h

```diff
@@ -306,6 +306,10 @@ struct NetworkAddressList {
 		return address.toString() + ", " + secondaryAddress.get().toString();
 	}
 
+	bool contain(const NetworkAddress& r) const {
```

Comment:
I think in cpp we usually name boolean functions like `contains`

Reply:
SG. Done.